compiler/repl-service/server/src/main/scala/com/digitalasset/daml/lf/ReplServiceMain.scala

```diff
@@ -155,8 +157,8 @@ object ReplServiceMain extends App {
       .addService(new ReplService(clients, timeMode, ec, sequencer, materializer))
       .maxInboundMessageSize(maxMessageSize)
       .build
-  server.start()
-  Files.write(config.portFile, Seq(server.getPort.toString).asJava)
+      .start
```

Comment:
I believe there isn’t actually a difference between the server returned by `start` and the one returned by `build` but this matches the grpc-java tutorial and avoids the warning.